wrlinux/wrlinux.go

```diff
@@ -0,0 +1,281 @@
+/*
+ * Copyright (c) 2022 Wind River Systems, Inc.
+ *
+ * The right to copy, distribute, modify, or otherwise make use
+ * of this software may be licensed only pursuant to the terms
+ * of an applicable Wind River license agreement.
+ */
+
+package wrlinux
+
+import (
+	"bufio"
+	"io"
+	"log"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/aquasecurity/vuln-list-update/git"
+	"github.com/aquasecurity/vuln-list-update/utils"
+	"github.com/araddon/dateparse"
+	"golang.org/x/xerrors"
+)
+
+const (
+	cveTrackerDir = "windriver-cve-tracker"
+	windriverDir  = "wrlinux"
+)
+
+var (
+	repoURLs = []string{
+		"https://distro.windriver.com/git/windriver-cve-tracker.git",
+	}
+	targets = []string{
+		"active",
+	}
+	statuses = []string{
+		"released",
+		"pending",
+		"not-affected",
+		"ignored",
+	}
+)
+
+type Vulnerability struct {
+	Candidate   string
+	PublicDate  time.Time
+	Description string
+	References  []string
+	Notes       []string
+	Priority    string
+	Bugs        []string
+	Patches     map[Package]Statuses
+}
+
+type Package string
+
+type Release string
+
+type Statuses map[Release]Status
+
+type Status struct {
+	Status string
+	Note   string
+}
+
+func Update() error {
+	var err error
+	gc := git.Config{}
+	dir := filepath.Join(utils.CacheDir(), cveTrackerDir)
+	for _, url := range repoURLs {
+		_, err = gc.CloneOrPull(url, dir, "master", false)
+		if err == nil {
+			break
+		}
+		log.Printf("failed to clone or pull: %s: %v", url, err)
+		log.Printf("removing %s directory", cveTrackerDir)
+		if err := os.RemoveAll(dir); err != nil {
+			return xerrors.Errorf("failed to remove %s directory: %w", cveTrackerDir, err)
+		}
```

Comment:
this is extra code

https://github.com/aquasecurity/vuln-list-update/blob/365b5e67a219f4eadd5ba0db13668a598862b3d0/wrlinux/wrlinux.go#L86

this is enough

Reply:
correct and addressed.